src/utils/request.js

```diff
@@ -133,26 +133,32 @@ export default function request(url, options) {
 
   showLoading &&
     window.g_app._store.dispatch({
-      type: "global/showLoading"
+      type: 'global/showLoading'
     });
   return axios(newOptions)
     .then(checkStatus)
-    .then(response => {
+    .then((response) => {
       showLoading &&
         window.g_app._store.dispatch({
-          type: "global/hiddenLoading"
+          type: 'global/hiddenLoading'
         });
-
-      const res = response.data.data || {};
+      let res = response.data.data || {};
```

Comment:
It is more appropriate to use a parameter to control the return value.

Reply:
Have to repair